test/files/ALS/als016/fail-als016-scenario02-imperial_break_in_direction_angle.ifc

```diff
@@ -1,263 +1,447 @@
 ISO-10303-21;
 HEADER;
-FILE_DESCRIPTION(('ViewDefinition [ReferenceView]'),'2;1');
-FILE_NAME('fail-als016-scenario02-imperial_break_in_direction_angle.ifc','2024-05-09T17:23:26',(''),(''),'','IfcOpenShell contributors - IfcOpenShell - v0.8.0+0d3c73de8','');
+FILE_DESCRIPTION(('ViewDefinition [CoordinationView]'),'2;1');
+FILE_NAME('','2025-06-19T13:13:25',(''),(''),'IfcOpenShell 0.8.3-2db6b86','IfcOpenShell 0.8.3-2db6b86','');
```

Comment:
```suggestion
FILE_DESCRIPTION(('ViewDefinition [ReferenceView]'),'2;1');
FILE_NAME('','2025-06-19T13:13:25',(''),(''),'IfcOpenShell contributors - IfcOpenShell - v0.8.3+2db6b86','');
```
The header policy of the file is failing (i.e. CoordinationView is not a valid MVD and the originating_system is not in the valid format)

![image](https://github.com/user-attachments/assets/29a0193f-7da3-4c44-8318-815ad4668592)